docs/contributing/pull-request-walkthrough.rst

```diff
@@ -0,0 +1,57 @@
+Pull Request Walkthrough
+========================
+
+Changes to pulpcore are submitted via `GitHub Pull Requests (PR)
+<https://help.github.com/articles/about-pull-requests/>`_ to the `pulp git repository
+<https://github.com/pulp/pulp>`_ . Plugin git repositories are listed in the :ref:`plugin
+table<plugin-table>`.
+
+Boilerplate
+-----------
+
+If you would like to submit a patch, especially if you have a major change, it is recommended that
+you :ref:`chat with us<community>` before you get started.
+
+#. `Fork <https://help.github.com/articles/fork-a-repo/>`_ `pulp <https://github.com/pulp/pulp>`_ in
+   your GitHub account.
+#. :doc:`Install Pulp from source.<dev-setup/index>`
+#. Create a new branch from the :ref:`appropriate base branch<git-branch>`
+#. Review the :doc:`style-guide`.
+
+Checklist
+---------
+
+#. Add :doc:`unit tests<unit_tests>` where appropriate.
+#. Update relevent :doc:`documentation`. Please build the docs to test!
+#. If your change would benefit from integration testing, write a `pulp smash issue
+   <https://github.com/PulpQE/pulp-smash/issues/new>`_.
+#. If you are adding a new feature, add a release note.
+#. :ref:`Rebase and squash<rebase>` to a single commit, if appropriate.
+#. Write an excellent :ref:`commit-message`. Make sure you reference and link to the issue.
+#. Push your branch to your fork and open a `Pull request across forks
+   <https://help.github.com/articles/creating-a-pull-request-from-a-fork/>`_.
+#. Add GitHub labels as appropriate.
+#. Change the status of the redmine issue to "POST".
+#. Add a link to the pull request in a comment on the issue.
+#. Set the "Target Release" field on the issue.
```

Comment:
"Target Release" is not documented in the Redmine Fields documentation up above.

Reply:
Nice catch!!